ckanext/vocabulary_services/logic/action/get.py

```diff
@@ -78,12 +78,14 @@ def csiro_vocabulary_terms(context, data_dict):
                         term = response['@graph'][i]
                         uri = term.get('@id', None)
                         label = term.get('rdfs:label', None)
+                        definition = term.get('dct:definition', None) or term.get('dct:description', None)
```

Comment:
@salsa-nathan Is the logic correct? If there is a definition, use the definition value and fallback to using description value if the definition does not exist.

Reply:
@MarkCalvert I think `term.get('dct:definition', None)` might need to be changed to `term.get('skos:definition', None)` based on what I see in the JSON coming from CSIRO, e.g.

Snippet from http://registry.it.csiro.au/def/isotc211/MD_ClassificationCode?_format=jsonld :
```
{
    "@id" : "http://registry.it.csiro.au/def/isotc211/MD_ClassificationCode/secret",
    "@type" : [ "skos:Concept", "http://def.isotc211.org/iso19115/-1/2014/ConstraintInformation#MD_ClassificationCode" ],
    "dct:description" : "kept or meant to be kept private, unknown, or hidden from all but a select group of people",
    "rdfs:isDefinedBy" : {
      "@id" : "http://standards.iso.org/iso/19115/-1/ed-1/en/"
    },
    "rdfs:label" : "secret",
    "skos:definition" : "kept or meant to be kept private, unknown, or hidden from all but a select group of people",
    "skos:inScheme" : {
      "@id" : "http://def.isotc211.org/iso19115/-1/2014/ConstraintInformation/code/MD_ClassificationCode"
    },
```

Reply:
Good spotting